HexCtrl/HexCtrl.h

```diff
@@ -74,6 +74,7 @@ namespace HEXCTRL
 		[[nodiscard]] virtual bool IsOffsetAsHex()const = 0;   //Is "Offset" currently represented (shown) as Hex or as Decimal.
 		[[nodiscard]] virtual auto IsOffsetVisible(ULONGLONG ullOffset)const->HEXVISION = 0; //Ensures that the given offset is visible.
 		[[nodiscard]] virtual bool IsVirtual()const = 0;       //Is working in Virtual or default mode.
+		[[nodiscard]] virtual DWORD GetCurrentDateFormat()const = 0;   //Determine current date format

```

Comment:
IMO GetCurrentDateFormat is too verbose. `GetDateFormat` / `SetDateFormat` is perfectly fine.
Also, by convention all public methods are alphabetically ordered.

Reply:
I have rename the methods as requested and alphabetically ordered.